addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -21,7 +21,7 @@
 class RecruitmentSource(models.Model):
     _name = "hr.recruitment.source"
     _description = "Source of Applicants"
-    _inherits = {"utm.source": "source_id"}
+    _inherit = ['utm.source.mixin']
 
     source_id = fields.Many2one('utm.source', "Source", ondelete='restrict', required=True)
```

Comment:
Seems we could remove the source_id field definition as it comes from the mixin ?